third_party/envoy-api.WORKSPACE

```diff
@@ -0,0 +1,16 @@
+workspace(name = "envoy_api")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
This is my only outstanding concern with this PR.

Reply:
Oops, I missed this comment when scrolling down the change list. We need to patch in a WORKSPACE file because they don't have a default one (https://github.com/envoyproxy/data-plane-api).

Reply:
Wow. That *is* surprising. It looks like they pull their WORKSPACE file in [from somewhere outside the source tree](https://github.com/envoyproxy/data-plane-api/blob/0026887e3204008527e357d47544a8b949bb8940/ci/build_setup.sh#L28).